src/main/python/tensorframes/core_test.py

```diff
@@ -198,7 +199,22 @@ def test_reduce_rows_1(self):
             res = tfs.reduce_rows(x, df)
         assert res == sum([r.x for r in data])
 
-    # This test fails
+    def test_append_shape(self):
+        data = [Row(x=float(x)) for x in range(5)]
+        df = self.sql.createDataFrame(data)
+        ddf = tfs.append_shape(df, col('x'), [-1, 1])
```

Comment:
Do you know what happens if you replace -1 by None?

Reply:
the py4j would be unhappy, so i can do a convert from None to -1 .
i miss the type annotation py3 have, so that i don't need to worry about it